cypress/commons/services/stubbing.js

```diff
@@ -26,6 +26,8 @@ const STUB_TYPES = [
   'PERMISSIONS_MAPPING',
   'UPDATE_DATASET',
   'UPDATE_SCENARIO',
+  'DELETE_DATASET',
+  'REFRESH_DATASET',
```

Comment:
We probably don't need to create custom stubbing types for _delete_ & _refresh_ events, we can enable the stubbing for these events when the stubbing is enabled for `CREATE_DATASET` :slightly_smiling_face: (it could help reduce the number of stubbing options to set when writing new tests)

Reply:
yes, you are right, I introduced `REFRESH_DATASET` to make a difference between refreshing alone and after creating, but finally, I made a special function for the first one, so no need to add an extra stubbing option :+1: